BoxOffice/Network/DataMapping/DailyBoxOfficeResponseDTO.swift

```diff
@@ -59,3 +59,14 @@ extension DailyBoxOfficeResponseDTO {
         let screenCount, showCount: String
     }
 }
+
+extension DailyBoxOfficeResponseDTO.DailyBoxOfficeDTO {
+    func toModel() -> BoxOffice {
+        return .init(rank: Int(rank)!,
+                     rankChange: Int(rankChange)!,
```

Comment:
확신이 있더라도 휴먼에러가 발생할 수도 있어서 !는 지양하는게 좋다고 생각합니다!
```suggestion
        return .init(rank: Int(rank) ?? .zero,
                     rankChange: Int(rankChange)  ?? .zero,
```